dataprofiler/data_readers/data_utils.py

```diff
@@ -303,6 +302,43 @@ def detect_file_encoding(file_path, buffer_size=1024, max_lines=20):
     # Typical file representation is utf-8 instead of ascii, treat as such.
     if not encoding or encoding == 'ascii':
```

Comment:
@lettergram here do we want to set the if statement for if windows-1254 set to 'utf-8'?

Reply:
we should also add the data that fails if this isn't true to check for it

Reply:
It might be a better idea to just leave it to CnM since I would assume there could be a file that actually does have the windows-1254 encoding (or alternatively put it in the except if CnM is not installed).

Reply:
i think the idea is that the error was common enough that we should do that by default as a known error of the library and then CnM will correct it if we actually are incorrect.

Reply:
I used the alternative solution in the new commit (and changed the test accordingly)

Reply:
@JGSweets, I can swap it and if windows-1254 is actually valid then CnM will correct it back to windows-1254 (given it is installed)